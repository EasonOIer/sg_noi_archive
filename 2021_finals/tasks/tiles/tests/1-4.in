8 8
x.x...x.
.x......
xx.x.x..
2 2 8
1 3 2
2 2 8
2 2 8
1 3 5
2 1 8
2 2 8
1 1 4
