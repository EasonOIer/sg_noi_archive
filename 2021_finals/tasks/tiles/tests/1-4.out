935
1495
1495
1050
1050
